neo/rawio/spikeglxrawio.py

```diff
@@ -396,52 +396,60 @@ def parse_spikeglx_fname(fname):
     This function is copied/modified from Graham Findlay.
 
     Notes:
-       * Sometimes the original file name is modified by the user and "_gt0_" or "_t0_"
+        * Sometimes the original file name is modified by the user and "_gt0_" or "_t0_"
           are manually removed. In that case gate_name and trigger_num will be None.
+        * If tcat is used, then the trigger_num will be set to "cat".
 
     Parameters
     ---------
     fname: str
```

Comment:
We probably need to actually change this since this required the dir to run with the tcat format. We should at least explain this in the docstring.

Reply:
@zm711 I don't understand what you mean here. Can you elaborate?

Reply:
anyways, I gave it a go to improve it ;)

Reply:
Sorry I was offline for the 4th. I mean the `fname`. When I was testing the code it would load the catGT files when I would do the parent directory instead of the actual file. So current the name says it requires filename without the extension, but in some cases it also requires a directory? I'm heading to lab now so I can test this on my laptop there to doublecheck.

Reply:
no I think this is correct. The `parse_spikeglx_fname` is called for every `.meta` file using the stem, so I think that the name `fname` and the description are fine!